crt_portal/cts_forms/views.py

```diff
@@ -222,6 +222,16 @@ def setup_filter_parameters(report, querydict):
 
 @login_required
 def IndexView(request):
+    profile_form = ProfileForm()
+
+    # Check for Profile object, then add filter to request
+    if hasattr(request.user, 'profile') and request.user.profile.intake_filters:
+        request.GET = request.GET.copy()
+        request.GET.setlist('assigned_section', request.user.profile.intake_filters.split(','))
+
+        # Retreive ProfileForm intake_filters or return POST ProfileForm
+        profile_form.fields['intake_filters'].initial = request.user.profile.intake_filters.split(',')
```

Comment:
We should set the initial values for this form by using a bound instance of ProfileForm